boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -27,6 +27,34 @@
 			gpios = <&gpio0 28 0>, <&gpio0 29 0>, <&gpio0 30 0>;
 		};
 	};
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
```

Comment:
I checked all the pins. Would appreciate a word about the cells and map-mask.

Reply:
Word given, resolved